README.md

```diff
@@ -94,6 +97,7 @@ The default configuration directory is `$HOME/.config/anyrun` the structure of t
 ```
 
 The config file has the following structure, and as seen in the name uses the `ron` language:
+
 ```ron
 Config(
   width: 800, // The width of the window
```

Comment:
This is also missing the new configuration options.

Reply:
fixed.